examples/tv-casting-app/APIs.md

```diff
@@ -1,123 +1,155 @@
 
 # Matter Casting APIs
 
-  
+Matter Casting consists of three parts:
+
+* **The mobile app**: For most conent providers, this would be your consumer-facing mobile app.  By adding a Matter "Casting Client" to your mobile app, you enable the user to discover casting targets, cast content, and control casting sessions.
+* **The TV content app**: For most conent providers, this would be your consumer-facing app on a Smart TV.  By enhancing your TV app to act as a Matter "Casting Player", you enable Matter Casting Clients like your mobile app to cast content. The [example Matter tv-casting-app](https://github.com/project-chip/connectedhomeip/tree/master/examples/tv-casting-app) for Android / iOS and Linux builds on top of the Matter SDK to demonstrate how a TV app works.
```

Comment:
```suggestion
* **The TV content app**: For most content providers, this would be your consumer-facing app on a Smart TV.  By enhancing your TV app to act as a Matter "Content app", you enable Matter Casting Clients to cast content. The [example Matter content-app](https://github.com/project-chip/connectedhomeip/tree/master/examples/tv-app/android/App/content-app) for Android builds on top of the Matter SDK to demonstrate how a TV Content app works.
```